modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Panel/PanelHeader.tsx

```diff
@@ -54,17 +54,6 @@ export function PanelHeader({
 					}
 				)}
 			>
-				{type === 'regular' && (
-					<ClayButtonWithIcon
-						aria-label={Liferay.Util.sub(
-							Liferay.Language.get('drag-x'),
-							title
-						)}
-						displayType="unstyled"
-						symbol="drag"
-					/>
-				)}
-
```

Comment:
@diogomelo09, we have to adjust the style here to, you can use the same `margin-left: 1rem;` but  to the `object-admin-panel__header__content-left `class in `Panel.scss`.

Reply:
Just did it!! :tada: